cmake/modules/MacOSXBundleInfo.plist.in

```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleIconFile</key>
 	<string>flameshot</string>
 	<key>CFBundleIdentifier</key>
-	<string>https://flameshot.org/</string>
+	<string>org.flameshot</string>
```

Comment:
@borgmanJeremy Is this good?

Reply:
I'm not very familiar with MacOs packaging actually, it was contributed by the community.

Reply:
I checked the specification and yes this change is correct: https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102070

Reply:
@borgmanJeremy Just did some googling, and I think this is good. Source: [this](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleidentifier) + looked at some reputable projects that support Mac.

Reply:
Looks like we ended up in the same documentation :) .

Reply:
Xcode also complains that the bundle ID from .plist file should be in sync with PRODUCT_BUNDLE_IDENTIFIER in Xcode project settings (flameshot.xcodeproj), which is empty right now. It looks like this [can be achieved](https://cmake.org/cmake/help/v3.4/prop_tgt/XCODE_ATTRIBUTE_an-attribute.html) by setting a property named `XCODE_ATTRIBUTE_PRODUCT_BUNDLE_IDENTIFIER` on the flameshot target in CMakeLists.txt. Do you think this also need to be fixed?

Reply:
@sryze If it complains, then I think it's better to set the property.

While we are on the topic of MacOS,  could you make [these version strings](https://github.com/flameshot-org/flameshot/blob/f214757c7bc34b6e6e7b7c299f0e52d15fa0af97/cmake/modules/MacOSXBundleInfo.plist.in#L25-L28) be automatically taken from the CMake configuration, so we don't have to change the plist file manually for each new release? The file has a `.in` extension, so I'm kind of assuming that it is preprocessed by something (hopefully CMake) before the final `Info.plist` is created?

Reply:
Yep, it is pre-processed by CMake. Added the version string there.

Reply:
@sryze Perfect, thanks.